packages/spartan-council/.env.sample

```diff
@@ -1,11 +1 @@
-# -------------------------------------------------------------------------------
-# PRIVATE KEY
-# -------------------------------------------------------------------------------
-DEPLOYER_PRIVATE_KEY=
-# -------------------------------------------------------------------------------
-
-# -------------------------------------------------------------------------------
-# INFURA PROVIDER
-# -------------------------------------------------------------------------------
-INFURA_KEY=
-# -------------------------------------------------------------------------------
+FORK_PROVIDER_URL=https://opt-mainnet.g.alchemy.com/v2/<api-key>
```

Comment:
is this intentional?

Reply:
Yup, we need that env var for executing integration tests, and we don't need the previously declared vars

Reply:
Sorry, you were right, we need `DEPLOYER_PRIVATE_KEY`, re adding

Reply:
👌